autopush/integration/test/index.html

```diff
@@ -29,17 +31,17 @@
   </script>
 
   <!-- test dependencies -->
-  <script src="./vendor/mocha-2.2.5.js"></script>
-  <script src="./vendor/chai-3.0.0.js"></script>
-  <!-- <script src="../node_modules/chai/chai.js"></script>-->
-  <!-- <script src="./vendor/sinon-1.16.1.js"></script> -->
-  <!-- <script src="./sdk_mock.js"></script> -->
+  <script src="vendor/mocha-2.2.5.js"></script>
+  <script src="vendor/chai-3.0.0.js"></script>
   <script>
     chai.config.includeStack = true;
     mocha.setup({ui: 'bdd', timeout: 10000});
   </script>
+
   <!-- Test scripts -->
+  <script src="/app/converter.js"></script>
```

Comment:
Added link to "app/converter.js" to bring in our lib to test. This will make `converter` available as a global var.